### DISCUSSION TITLE
Does Firefly II cron handle missed recurring transactions?

### DISCUSSION THREAD

**Discussion by elliot-gh** [firefly-iii/firefly-iii#5595] [Firefly III] [answered]
I self host the Docker image and had to temporarily turn off the server as I was moving. I finally got to turn on my server, and it seems none of the existing recurring transactions that were set to happen in the days the server were off didn't "catch up" and get created? I even manually called the cron API to double check this was getting run. I did have a time zone change (UTC-7 -> UTC-8) if that matters. I was on 5.6.5 at the first boot (and noticed this behavior), and later upgraded to 5.6.10.

**Comment by JC5** [comment, marked as answer]
No, it doesn't recreate missing instances.
